docs/src/man/split_apply_combine.md

```diff
@@ -114,9 +175,8 @@ julia> by(iris, :Species, nrow, :PetalLength => mean => :mean)
 │ 2   │ Iris-versicolor │ 50    │ 4.26    │
 │ 3   │ Iris-virginica  │ 50    │ 5.552   │
 
-julia> by(iris, :Species,
-          [:PetalLength, :SepalLength] =>
-          (p, s) -> (a=mean(p)/mean(s), b=sum(p))) # multiple columns are passed as arguments
+julia> combine([:PetalLength, :SepalLength] => (p, s) -> (a=mean(p)/mean(s), b=sum(p)),
```

Comment:
Are we deprecating `combine(fun, gdf)`? If not, we should add an example showing how this call is equivelent to a function syntax. 

I have no strong feelings about the deprecation, but wouldn't mind a `DF()` column selector to be added the future.

Reply:
We are not deprecating it. The example you ask for is given in the docstring of combine. I will add it also here as you requested.